container/docker/handler.go

```diff
@@ -168,7 +168,8 @@ func newDockerContainerHandler(
 		ignoreMetrics:      ignoreMetrics,
 	}
 
-	if !ignoreMetrics.Has(container.DiskUsageMetrics) {
+	if !ignoreMetrics.Has(container.DiskUsageMetrics) && storageDriver != devicemapperStorageDriver {
```

Comment:
Yeah. `rootfsStorageDir` is storage driver dependent.

On Mon, Feb 29, 2016 at 2:58 PM, Seth Jennings <notifications@github.com>
wrote:

> In container/docker/handler.go
> <https://github.com/google/cadvisor/pull/1143#discussion_r54494773>:
>
> > @@ -168,7 +168,8 @@ func newDockerContainerHandler(
> >  		ignoreMetrics:      ignoreMetrics,
> >  	}
> >
> > -	if !ignoreMetrics.Has(container.DiskUsageMetrics) {
> > +	if !ignoreMetrics.Has(container.DiskUsageMetrics) && storageDriver != devicemapperStorageDriver {
>
> You are saying that we should only not gather stats for the
> rootfsStorageDir?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/1143/files#r54494773>.
>